packages/network/src/chain/Chain.ts

```diff
@@ -173,10 +174,20 @@ export async function signAndSubmitTx(
   tx: SubmittableExtrinsic,
   signer: KeyringPair,
   {
-    nonce = -1,
+    tip,
```

Comment:
tip?

Reply:
Because of this error - "error TS2339: Property 'nonce' does not exist on type 'Partial<Options> & Partial<{ tip: AnyNumber; }>'."